ci/Jenkinsfile.android

```diff
@@ -0,0 +1,72 @@
+pipeline {
+  agent { label 'macos' }
+
+  options {
+    buildDiscarder(logRotator(
+      numToKeepStr: '10',
+      daysToKeepStr: '30',
+      artifactNumToKeepStr: '4',
+    ))
+  }
+  
+  environment {
+    LANG = 'en_US.UTF-8'
+    LANGUAGE = 'en_US.UTF-8'
+    LC_ALL = 'en_US.UTF-8'
+    FASTLANE_DISABLE_COLORS = 1
+    REALM_DISABLE_ANALYTICS = 1
+    ANDROID_HOME = '/usr/local/share/android-sdk'
+    ANDROID_SDK_ROOT = '/usr/local/share/android-sdk'
+    ANDROID_NDK = '/Users/jenkins/android-ndk-r10e'
+    ANDROID_NDK_HOME = '/Users/jenkins/android-ndk-r10e'
+  }
+  
+  stages {
+    stage('Prep') {
+      steps {
+        script {
+          /* Necessary to load methods */
+          mobile = load 'ci/mobile.groovy'
```

Comment:
Can we do that earlier in the code? We use `mobile` in other stages but declaration and loading happens here.

Reply:
What do you mean? This is the first reference to `mobile` in that file.

Reply:
It was a bit misleading to me that `mobile` is used other stages while it looks like it is loaded only for `stage('Prep')` but if it works this way than fine.

Reply:
The trick is that I'm not using `def` for `mobile` variable which makes it global variable available in other stages.